cluster/examples/kubernetes/ceph/cluster-on-pvc.yaml

```diff
@@ -23,3 +31,44 @@ spec:
         resources:
           requests:
             storage: 10Gi
+  cephVersion:
+    image: ceph/ceph:v14.2.1-20190430
+    allowUnsupported: false
+  dashboard:
+    enabled: true
+  network:
+    hostNetwork: false
+  storage:
+    storageClassDeviceSets:
+    - name: cluster1-set1
+      count: 3
+      resources:
+        requests:
+          cpu: 2
+          memory: 4Gi
+      # placement:
+      #   podAntiAffinity:
+      #     preferredDuringSchedulingIgnoredDuringExecution:
+      #     - weight: 100
+      #       podAffinityTerm:
+      #         labelSelector:
+      #           matchExpressions:
+      #           - key: "rook.io/cluster"
+      #             operator: In
+      #             values:
+      #               - cluster1
+      #           topologyKey: "failure-domain.beta.kubernetes.io/zone"
+      volumeClaimTemplates:
+      - metadata:
+          name: template-1
```

Comment:
Yes, the template name should be required. Otherwise there's no means to properly identify what set a device belongs to since we can't rely on simple ordering of the list.

Reply:
We really only need to require a unique name for the templates. The blank name could be valid. But we can save the discussion for another day, I wouldn't block this PR on it.

Reply:
it will work without the template name as well as the code handles it. But it will good if the name is provided.